internal/api/utils.go

```diff
@@ -194,29 +194,15 @@ func get_user_permissions(c echo.Context) map[string][]string {
 	return user_permissions
 }
 
-func transformComponentUnits(recommendationJSON map[string]interface{}) map[string]interface{} {
+func truncateComponentUnits(recommendationJSON map[string]interface{}) map[string]interface{} {
 	/*
-		Converts units for Memory and CPU
-		bytes -> MiB -> GiB
-		cores -> millicores
+		Truncates CPU units(cores) to three decimal places
+		Truncates Memory units(Mi) to two decimal places
+		Hack: Truncates duration_in_hours to one decimal places
+		To Do: Once Kruize returns identical values for duration_in_hours
```

Comment:
```suggestion
		TODO: Once Kruize returns identical values for duration_in_hours
```
Nitpick: Helpful to discover Todos across codebase by editor/IDE